lib/Jsrt/Jsrt.cpp

```diff
@@ -2697,24 +2699,31 @@ JsErrorCode RunSerializedScriptCore(const wchar_t *script, JsSerializedScriptLoa
     });
 }
 
+#ifdef _WIN32
+template
+JsErrorCode RunSerializedScriptCore<JsSerializedScriptLoadSourceCallback, JsSerializedScriptUnloadCallback>(const wchar_t *script, JsSerializedScriptLoadSourceCallback scriptLoadCallback,
+    JsSerializedScriptUnloadCallback scriptUnloadCallback, unsigned char *buffer, JsSourceContext sourceContext,
+    const wchar_t *sourceUrl, bool parseOnly, JsValueRef *result);
```

Comment:
Is this needed? Wouldn't the compiler automatically figure out needed template instantiations from the callers below?